docs/README.md

```diff
@@ -81,23 +79,21 @@ Find out more about configuring the TCP protocol on the [TCP configuration](netw
 - [Maven plugin (fuinorg/event-store-maven-plugin)](https://github.com/fuinorg/event-store-maven-plugin)
 - [Go (jdextraze/go-gesclient)](https://github.com/jdextraze/go-gesclient)
 - [PHP (prooph/event-store-client)](https://github.com/prooph/event-store-client/)
+- [JVM Client (EventStore/EventStore.JVM)](https://github.com/EventStore/EventStore.JVM)
+- [Haskell (EventStore/EventStoreDB-Client-Haskell)](https://github.com/EventStore/EventStoreDB-Client-Haskell)
```

Comment:
Should we make a note that all those are TCP based and will not work with 23.10 onwards ?

Reply:
I feel like that is covered by the deprecation note which applies to this whole TCP section, but I am happy to add another note in this "community supported clients" section as well. It may be a good place to also explain that community supported clients are developed and maintained by community members, not Event Store.